### PR TITLE
Add Genre and language article

### PR DESCRIPTION
**Pull request by cl8n**
nat meeting said a description of the genres would be helpful

### CODE REVIEW

File: wiki/Beatmaps/Genre_and_language/en.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - genres
+  - languages
+  - metadata
+---
+
+# Genre and language
+
+![A screenshot of Source, Genre, Language, and Tags on a beatmap page](img/beatmap-info.png "Genre and language are listed on beatmap pages.")
+
+[Beatmaps](/wiki/Beatmaps) submitted to the website have **genre** and **language** fields to help categorise songs. They default to `Unspecified` and `Other` respectively, and can be edited by mapset owners, [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) members, and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members.
```
Comment (Nivalyx) [inline wiki/Beatmaps/Genre_and_language/en.md:12]: might want to add a detail which explains that once a map got Ranked/Loved the mapset owner loses their power to modify the Genre and Language settings in their beatmap

something like

> (...) and can be edited by [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members. Furthermore, mapset owners are also able to set the genre and language of their own beatmaps as long as the said beatmap's status is either WIP, Pending, or Graveyarded.

**Comment by cl8n** [comment]
@yaspo @dorothy3242 if u have time could u look over this one, I'm not sure if it fits the purpose u intended from that NAT meeting

File: wiki/Beatmaps/Genre_and_language/en.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - genres
+  - languages
+  - metadata
+---
+
+# Genre and language
+
+![A screenshot of source, genre, language, and tags on a beatmap page](img/beatmap-info.png "Genre and language are listed on beatmap pages.")
```
Comment (yaspo) [inline wiki/Beatmaps/Genre_and_language/en.md:10]: Could be better to use a screenshot with an actual language like English etc. in it imo, having "Instrumental" on the cover for languages seems a bit off

File: wiki/Beatmaps/Genre_and_language/en.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Beatmap listing
+
+![The available filters on the beatmap listing](img/beatmap-listing.png)
+
+The [beatmap listing](https://osu.ppy.sh/beatmapsets) has filters available for genre and language in the `More Search Options` dropdown.
+
+## List of genres
+
+| Genre | Traits of applicable songs |
```
Comment (yaspo) [inline wiki/Beatmaps/Genre_and_language/en.md:24]: Think a more natural order could be used for this list, regular users can't see the dropdown for genres.
Would move non-genres to the bottom and keep Unspecified + Other, Anime + Video Game next to each other.

File: wiki/Beatmaps/Genre_and_language/en.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+| :-- | :-- |
+| Unspecified | No songs are applicable. This genre is a placeholder until a proper one is set. |
+| Video Game | Made for or made popular by a video game. This is also used for arrangements and remixes of those songs. |
+| Anime | Made for or made popular by an anime or similar medium. |
+| Rock | Mostly focusing on guitars, drums, and bass, typically "built on a foundation of simple unsyncopated rhythms" and characterised by "live performance and a focus on serious and progressive themes" ([*Rock music* on Wikipedia](https://en.wikipedia.org/wiki/Rock_music#Characteristics)). |
+| Pop | Prominent in pop culture, and composed to be catchy by use of simple structure and repeated choruses or melodies. |
+| Other | Not belonging in the other genres. |
```
Comment (yaspo) [inline wiki/Beatmaps/Genre_and_language/en.md:31]: Could make this "Not belonging to any of the listed genres" ? Is more specific to this listing that way imo

File: wiki/Beatmaps/Genre_and_language/en.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- Chinese
+- French
+- German
+- Italian
+- Japanese
+- Korean
+- Spanish
+- Swedish
+- Instrumental
+- Other (including songs with many prominent languages)
```
Comment (yaspo) [inline wiki/Beatmaps/Genre_and_language/en.md:48]: many -> multiple, something evenly split between just 2 languages would probably be categorized here too, so using 'many' seems off

File: wiki/Beatmaps/Genre_and_language/en.md (hunk 3 above)
Comment (dorothy3242) [reply wiki/Beatmaps/Genre_and_language/en.md:24]: Eh, it's the same order that's used on the website for the search filter, so the current way seems fitting enough

File: wiki/Beatmaps/Genre_and_language/en.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  - genres
+  - languages
+  - metadata
+---
+
+# Genre and language
+
+![A screenshot of source, genre, language, and tags on a beatmap page](img/beatmap-info.png "Genre and language are listed on beatmap pages.")
+
+[Beatmaps](/wiki/Beatmaps) submitted to the website have **genre** and **language** fields to help categorise songs. They default to `Unspecified` and `Other` respectively, and can be edited by [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members. Mapset owners are also able to set the genre and language of their beatmaps as long as the beatmap's status is WIP<!-- TODO: link -->, [Pending](/wiki/Beatmaps#pending), or [Graveyard](/wiki/Beatmaps#graveyard).
```
Comment (dorothy3242) [inline wiki/Beatmaps/Genre_and_language/en.md:12]:
```suggestion
[Beatmaps](/wiki/Beatmaps) submitted to the website have **genre** and **language** fields to help categorise songs. They default to `Unspecified` and `Other` respectively, and can be edited by [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members. Mapset owners are also able to set the genre and language of their beatmaps as long as the beatmap is in the WIP<!-- TODO: link -->, [Pending](/wiki/Beatmaps#pending), or [Graveyard](/wiki/Beatmaps#graveyard) categories.
```

the website refers to pending/graveyard/wip etc as Categories so this should be kept consistent.

File: wiki/Beatmaps/Genre_and_language/en.md (hunk 3 above)
Comment (cl8n) [reply wiki/Beatmaps/Genre_and_language/en.md:24]: not changed^

**Pull request merged**